integration_tests/requirements.txt

```diff
@@ -1,6 +1,7 @@
-dbt-snowflake~=1.0.0
-dbt-bigquery~=1.0.0
-dbt-redshift~=1.0.0
-dbt-postgres~=1.0.0
-dbt-spark~=1.0.0
-dbt-spark[PyHive]~=1.0.0
+dbt-snowflake>=1.0.0,<2.0.0
+dbt-bigquery>=1.0.0,<2.0.0
+dbt-redshift>=1.0.0,<2.0.0
```

Comment:
I don't think it is too much of a risk with us having the upper bound being 2.0.0 since this is only for integration testing and will not be used once we push the migration updates. However, for consistency I feel it makes sense to adjust. Updated.